app/server/logging/jsondb/src/test/resources/logs-controller-db.json

```diff
@@ -2,87 +2,9 @@
-          "pod" : "test-pod-x23x",
-          "status" : "done",
-          "steps" : {
-            "s2" : {
-              "-L38cZ5Jd1L876xV4vEG" : {
-                "duration" : 582977,
-                "logts" : "2018-01-12T21:22:02.068338027Z"
-              },
-              "-L38cZ5Jd1L876xV4vEH" : {
-                "logts" : "2018-01-12T21:22:02.068338027Z",
-                "message" : "Hello World"
-              }
-            },
-            "s3" : {
-              "-L38cZ5Nd1L876xV4vEI" : {
-                "duration" : 18087,
-                "logts" : "2018-01-12T21:22:02.068338027Z"
-              }
-            },
-            "s4" : {
-              "-L38cZ5Od1L876xV4vEJ" : {
-                "duration" : 494949,
-                "logts" : "2018-01-12T21:22:02.068338027Z"
-              }
-            }
-          },
-          "ver" : "3"
-        },
-        "-L38cZ5Pd1L876xV4vEL" : {
-          "failed" : true,
-          "logts" : "2018-01-12T21:22:02.068338027Z",
-          "pod" : "test-pod-x23x",
-          "status" : "done",
-          "steps" : {
-            "s2" : {
-              "-L38cZ5Pd1L876xV4vEM" : {
-                "duration" : 213613,
-                "logts" : "2018-01-12T21:22:02.068338027Z"
-              },
-              "-L38cZ5Pd1L876xV4vEN" : {
-                "logts" : "2018-01-12T21:22:02.068338027Z",
-                "message" : "Hello World"
-              }
-            }
-          },
-          "ver" : "3"
-        },
-        "-L38cZ5Ud1L876xV4vEQ" : {
-          "failed" : false,
-          "logts" : "2018-01-12T21:22:02.068338027Z",
-          "pod" : "test-pod-x23x",
-          "status" : "done",
-          "steps" : {
-            "s2" : {
-              "-L38cZ5Vd1L876xV4vER" : {
-                "duration" : 247312,
-                "logts" : "2018-01-12T21:22:02.068338027Z"
-              },
-              "-L38cZ5Vd1L876xV4vES" : {
-                "logts" : "2018-01-12T21:22:02.068338027Z",
-                "message" : "Hello World"
-              }
-            },
-            "s3" : {
-              "-L38cZ5Vd1L876xV4vET" : {
-                "duration" : 4242,
-                "logts" : "2018-01-12T21:22:02.068338027Z"
-              }
-            },
-            "s4" : {
-              "-L38cZ5Vd1L876xV4vEU" : {
-                "duration" : 70724,
-                "logts" : "2018-01-12T21:22:02.068338027Z"
-              }
-            }
-          },
-          "ver" : "3"
-        }
+        "-L38cZ51d1L876xV4vEF" : "{\"id\":\"-L38cZ51d1L876xV4vEF\",\"logts\":\"2018-01-12T21:22:02.068338027Z\",\"at\":1516285084034,\"pod\":\"test-pod-x23x\",\"ver\":\"3\",\"status\":\"done\",\"failed\":false,\"steps\":[{\"id\":\"-L38cZ5Nd1L876xV4vEI\",\"at\":1516285084056,\"duration\":18087},{\"id\":\"-L38cZ5Od1L876xV4vEJ\",\"at\":1516285084057,\"duration\":494949},{\"id\":\"-L38cZ5Jd1L876xV4vEH\",\"at\":1516285084052,\"duration\":582977,\"messages\":[\"Hello World\"]}]}",
```

Comment:
dont think this will work with the latest change to the key generator.

btw, why is the value not an oject but an escaped JSON object as string. Any paraticular reason ?

Reply:
Sorry, thats just a test fixture independent of the key generator, so should be save. but maybe we change it nevertheless to the real key format.